inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -13,20 +25,19 @@ class StellaratorCoils(Scenario):
     coils can also be created before creating the StellaratorCoils object. 
 
     Attributes:
-        num_coils (int): The number of coils per field period 
-        (independent coils).
         coils (list): List of `Coil` objects.
         num_field_periods (int): Number of magnetic field periods.
-        Refers to the number of complete magnetic field repetitions 
-        within a stellarator. Represents how many times the magnetic
-        field pattern repeats itself along the toroidal direction.
+          Refers to the number of complete magnetic field repetitions 
+          within a stellarator. Represents how many times the magnetic
+          field pattern repeats itself along the toroidal direction.
```

Comment:
Do you mind adding a bit more info here? What is the typical number? 
Try to be a bit more pedagogic, and maybe point to the literature (here or in the class description).